zendesk_attachment_backup.py

```diff
@@ -69,37 +73,51 @@ def collect_urls(input_files, print_urls, **kwargs):
 
    # Print URLs to file if requested
    if print_urls:
-      with open('attachment_urls.txt', "w") as fh:
+      target_file=os.path.join(kwargs.get("output_dir"), 'attachment_urls.txt')
```

Comment:
BTW, I don't write 79 chars either, because I don't think [Centus Tabulator](http://www.columbia.edu/cu/computinghistory/census-tabulator.html) and evolved IBM punch card is any relevant today. Even PEP8 says you can go up to 120 chars, once agreed (which exactly I do to avoid endless scrolls). However, other things makes perfect sense, IMO.